index.html

```diff
@@ -0,0 +1,131 @@
+            </div>
+            <div class="two"></div>
+            <div class="three">
+                SKILLS
+            </div>
+            <div class="four"></div>
+            <div class="five">
+                EXPERTISE
+            </div>
+        </div>
+
+
+        <div class="flex3">
+                <div class="flex31">
+                    <ul><li>Bachlore degree in Computer science from New York </li>
+                        <li>High School Passed with 4.5</li>
+                        <li>Junior High school 3.5</li>
+                    </ul>
+                </div>
+
+                <div class="flex32">
+                    <ul>
+                    <li>api DEVELOPER</li>
+                    <li>FRONT END DEV</li>
+                    <li> TESTER </li>
+                    <li>UI/UX DEVELOPER</li>
+                    <li>BECK END DEVELOPER</li>
+                </ul>
+                </div>
+                <div class="flex33">
+                    <ul>
+                        <li>PRODUCT DESIGN</li>
+                        <li>SOCIAL MEDIA</li>
+                        <li>QUALITY CONTROL</li>
+                        <li>OPTIMIZATION</li>
+                        <li>MINIMAL DESIGN</li>
+                        </ul>
+                </div>
+        </div>
+
+        <div class="btDiv">
+                <div class="bone"></div>
+                <div class="btwo">
+                    EXPERIENCE
+                </div>
+                <div class="bthree"></div>
+            </div>
+
+        <div class="flex4">
+                <div class="flex41">
+                    <h4>INTERIOR DESIGNER</h4>
+                        <ul>
+                            <li>Distingush rooms and designd staff</li>
+                            <li>Bachlore degree in Interior Design</li>
+                            <li>Developed many website in career</li>
+                        </ul>
+                </div>
+                <div class="flex42"></div>
+
+                <div class="flex43">
+                    <div class="smallBox">
+                        <h6>NEW YORK</h6>
+                        <p>San Fransisco</p>
+                    </div>
+                </div>
+        </div>
+        <div class="flex5">
+                <div class="flex51">
+                        <h4>EXTERIOR DEVELOPER</h4>
+                        <ul>
+                            <li>Distingush rooms and designd staff</li>
+                            <li>Bachlore degree in Interior Design</li>
+                            <li>Developed many website in career</li>
+                        </ul>
+                </div>
+                <div class="flex52"></div>
+                <div class="flex53">
+                        <div class="smallBox">
+                            <h6>NEW YORK</h6>
+                             <p>San Fransisco</p>
+                        </div>
+                </div>
+        </div>
+        <div class="flex6">
```

Comment:
Use 'id' instead of class whenever there's a unique entity. Again comment applicable throughout.

Reply:
Done